src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -765,6 +765,9 @@ class PumpEquipmentDescription:
     esp_thermal_efficiency_model = attrib_enum(
         default=constants.PumpThermalEfficiencyModel.Constant
     )
+    enabled_density_correction: bool = attr.ib(
+        default=False, validator=instance_of(bool)
+    )
```

Comment:
Why `enabled_density_correction`?
Probably `enable_density_correction` is better.

Reply:
Ok. i change this.